backend/dal/Services/ActivityService.cs

```diff
@@ -0,0 +1,79 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Pims.Dal.Constants;
+using Pims.Dal.Entities;
+using Pims.Dal.Entities.Models;
+using Pims.Dal.Helpers;
+using Pims.Dal.Helpers.Extensions;
+using Pims.Dal.Repositories;
+using Pims.Dal.Security;
+
+namespace Pims.Dal.Services
+{
+    public class ActivityService : IActivityService
+    {
+        private readonly ClaimsPrincipal _user;
+        private readonly ILogger _logger;
+        private readonly IActivityRepository _activityRepository;
+        private readonly IActivityTemplateRepository _activityTemplateRepository;
+
+
+
+        public ActivityService(
+            ClaimsPrincipal user,
+            ILogger<ActivityService> logger,
+            IActivityRepository activityRepository,
+             IActivityTemplateRepository activityTemplateRepository,
+        IPropertyRepository propertyRepository,
+            ICoordinateTransformService coordinateService)
+        {
+            _user = user;
+            _logger = logger;
+            _activityRepository = activityRepository;
+            _activityTemplateRepository = activityTemplateRepository;
+        }
+
+        public PimsActivityInstance GetById(long id)
+        {
+            _logger.LogInformation("Getting activity with id {id}", id);
```

Comment:
Info might be too verbose for this type of logging

Reply:
shortened